src/generic_ui/scripts/ui.ts

```diff
@@ -134,6 +134,25 @@ export interface NotificationData {
   unique ?:string;
 }
 
+var PROXYING_SESSION_ID_LENGTH = 16;
+
```

Comment:
Why does this need to be coming from ui? Can't you generate it in remote-connection, before attempting to to start proxying?